src/main/scala/com/codacy/client/stash/service/PullRequestServices.scala

```diff
@@ -21,13 +21,21 @@ class PullRequestServices(client: StashClient) {
       direction: String = "INCOMING",
       state: String = "OPEN",
       order: String = "NEWEST",
-      includeAvatar: Boolean = false
+      includeAvatar: Boolean = false,
+      pageRequest: Option[PageRequest]
```

Comment:
Can you have non optional parameters after parameters with default values?
Isn't it better if the mandatory parameters come first?
Also keep in mind that this change breaks binary compatibility, so add a `:breaking:`

Reply:
Mandatory things should be before parameters with default values, if not the point of using default values kinda disappears a bit.